Code/Core/FileIO/PathUtils.cpp

```diff
@@ -290,4 +294,21 @@
     outRelativeFileName += pathB;
 }
 
+// GetFullPath
+//------------------------------------------------------------------------------
+/*static*/ AString PathUtils::GetFullPath( const AString & fileName )
```

Comment:
FASTBuild uses an output param style for returning complex types (rather than relying on return by value optimizations)
i.e.
```
void PathUtils::GetFullPath( const AString & filename, AString & outFilename )
```

This also allows use of stack types like AStackString to eliminate memory allocations.

(Note that I think we'd probably want to try and eliminate the need for this function entirely if possible, avoiding the portability issue of the unimplemented TODO)